addons/web_unsplash/static/src/scss/unsplash.scss

```diff
@@ -1,46 +1,30 @@
-.unsplash_img_container {
-    $o-border-image: 2px;
-    .o_unsplash_img_cell {
-        .o_image {
-            cursor: pointer;
-            transition: opacity 400ms;
-
-            &.o_webimage {
-                width: 100%;
-                height: auto;
-                background-image: none;
-
-                img {
-                    max-height: 150px;
-                    margin: 0 auto;
-                    border: $o-border-image solid transparent;
-                }
-            }
-
-        }
-        &.o_selected .o_image img {
-            border-color: $o-brand-primary;
-            box-shadow: 0px 0px 2px 2px $o-brand-secondary;
-        }
-        .author_name {
-            font-size: 11px;
+.o_unsplash_attachment_cell {
+    cursor: pointer;
+    &:hover {
+        .o_we_attachment_highlight {
+            border-color: $card-border-color;
+            box-shadow: 0px 0px 2px 2px $card-border-color;
```

Comment:
Don't use bootstrap variable in editor scss, it is dependent of the website theme, which is not what we want (we cannot counter all theme side effects but this one is an easy one).

Reply:
It's already used by both the regular media dialog, and since I need those to match I just copied the code over. I'm still thinking of a way to do this whole thing without code duplication though, wanted to do that and move the 192px inline style there at the same time, I'm just not sure how just yet. Should I replace `$card-border-color` by a hard-coded value? Something else?

Reply:
Ideally a variable defined in the web_editor.variables file... but you can skip that part if it is confusing, I will review the whole design later anyway.